api.bs

```diff
@@ -2016,6 +2026,201 @@ To <dfn noexport>parse a `Save-Impression` header</dfn> given a [=header value=]
 
 </div>
 
+## Measuring Conversions ## {#http-api-conversions}
+
+\`<dfn http-header><code>Measure-Conversion</code></dfn>\` is a
+[=structured header/dictionary|Dictionary Structured Header=]
+set on a response requesting that the user agent invoke the
+<a method for=Attribution>measureConversion()</a> API.
```

Comment:
In line with RFC 9651 we should state:

* unknown dictionary keys are ignored
* unknown parameters on dictionary items are ignored (effectively, all parameters are ignored)